simplq/src/main/java/me/simplq/service/TokenService.java

```diff
@@ -18,24 +23,33 @@
 import me.simplq.exceptions.SQInternalServerException;
 import me.simplq.exceptions.SQInvalidRequestException;
 import me.simplq.service.smsService.SmsManager;
-import org.apache.commons.lang3.ObjectUtils;
-import org.springframework.stereotype.Service;
 
 @Service
-@RequiredArgsConstructor
 public class TokenService {
 
   private static final String TOKEN_CREATION_MESSAGE =
       "Hi %s,\n"
           + "You have been added to %s. Your token number is %d. You can know your status visiting"
           + " %s\n"
           + "Thanks for using simplq.me";
+  private static final String TOKEN_NOTIFICATION_MESSAGE =
+      "Hi, your wait for %s is over! You can proceed";
+  
   private final TokenRepository tokenRepository;
   private final QueueRepository queueRepository;
   private final SmsManager smsManager;
   private final LoggedInUserInfo loggedInUserInfo;
-  private static final String TOKEN_NOTIFICATION_MESSAGE =
-      "Hi, your wait for %s is over! You can proceed";
+
+  @Autowired
+  public TokenService(TokenRepository tokenRepository, QueueRepository queueRepository,
+        SmsManager smsManager, LoggedInUserInfo loggedInUserInfo) {
+
+          this.tokenRepository = tokenRepository;
+          this.queueRepository = queueRepository;
+          this.smsManager = smsManager;
+          this.loggedInUserInfo = loggedInUserInfo;
+    
+  }
```

Comment:
Let's remove these constructors as discussed above, they will be autogenerated by lombok.